book/localization.rst

```diff
@@ -70,6 +70,8 @@ For the developer the only touching points with localizations are the
 configuration and the eventual use of a language switcher on the homepage.
 For the language switcher the ``localizations`` variable delivered to the twig template
 can be used, which contains an associative array with the parameters ``locale``, ``url`` and ``country``.
+The currently active locale can be obtained from the underlying Symfony Request object with
+``app.request.locale``.
```

Comment:
We can use the `app.locale` for template may rendering in CLI context without a request object. That was added sometime ago by Symfony.